nepec/README.md

```diff
@@ -0,0 +1,2 @@
+# Noise-extended probabilistic error cancellation (NEPEC)
+This folder contains the code organized in Jupyter notebooks to reproduce the results of the paper *Noise-extended probabilistic error cencellation*.
```

Comment:
When we have the arXiv number/link we should add that here! For folks browsing, we could put the abstract of said paper if you think that would help.

Reply:
Thanks Sarah, indeed I was waiting for the link.
Good point about the abstract it can be useful and we a have a lot of space in the README.

Note: I set this as a Draft PR since notebooks are only partially executed. I need to fully re-execute the notebooks locally before merging.